src/components/NewThermometer/NewThermometer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  Wrapper,
+  ColorBlock,
+  ThermometerContainer,
+  Label,
+  InfoIcon,
+  InfoLink,
+} from './NewThermometer.style';
+import { Level } from 'common/level';
+import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
+
+const levels = [
+  Level.LOW,
+  Level.MEDIUM,
+  Level.HIGH,
+  Level.CRITICAL,
+  Level.SUPER_CRITICAL,
+];
+
+const items = levels.map(level => {
+  const summaryLevel = LOCATION_SUMMARY_LEVELS[level];
+  return {
+    label: summaryLevel.name, // for tooltip?
+    color: summaryLevel.color,
+  };
+});
+
+const ThermometerLabel: React.FC<{ label: string }> = ({ label }) => {
+  return (
+    <Label>
+      <strong>{label}</strong> risk
+    </Label>
+  );
+};
+
+const AboutLink: React.FC = () => {
+  return (
+    <InfoLink to="learn">
+      <span>
+        <InfoIcon />
+        About risk levels
+      </span>
+    </InfoLink>
+  );
+};
+
+const NewThermometer: React.FC = () => {
```

Comment:
How about `HorizontalThermometer`? A bit leery of naming components `New` because inevitably they end up not being new after some time

Reply:
yeah `NewThermometer` wasn't very innovative of me. `HorizontalThermometer` it is!